shared/oae/api/oae.api.authentication.js

```diff
@@ -319,10 +319,33 @@ define(['exports', 'jquery', 'oae.api.config', 'oae.api.i18n', 'oae.api.user', '
         });
     };
 
+    /**
+     * Remove a specific tenant from the cookie where we keep track of tenants the user logged in to
+     *
+     * @param  {String}   currentTenant   The current tenant the user is visiting
+     * @param  {Function} callback        Standard callback function
+     */
+    const removeLoggedInTenanciesCookie = exports.removeLoggedInTenanciesCookie = function(currentTenant, callback) {
+        // let's remove this tenant to the loggedin_tenancies cookie
+        let cookieName = 'loggedin_tenancies';
+        let loggedInTenancies = docCookies.getItem(cookieName);
```

Comment:
Where is `docCookies` coming from? I don't seem to find where we fetch the cookies.

Reply:
it's a small lib I've included in `/shared/vendor/js` and is being imported in `oae.core.js`

Reply:
Isn't it just `cookies` in oae.core.js?

Reply:
It is, but it exports a variable `docCookies`

Reply:
Righto.